static/css/logo.css

```diff
@@ -0,0 +1,45 @@
+
+     @font-face{
+        font-family: agencyr;
+        src : url(agencyr.ttf);
```

Comment:
That's not a correct way to refer to font URL since it relies upon the font existing in the client's PC. I would suggest you use a URL from google fonts or other free font site and use the direct link to TTF font here.

Reply:
i tried with URL ..but didnt work...and thanks for guiding...😊